features/gezag-minderjarige/1.1-ingezetene-BRP.feature

```diff
@@ -0,0 +1,268 @@
+      | type        | GezagNietTeBepalen                                                   |
+      | toelichting | gezag is niet te bepalen omdat minderjarige niet in Nederland woont. |
+
+
+  Regel: Als gemeente van inschrijving in onderzoek staat kan het gezag niet worden bepaald
+
+    Abstract Scenario: gezag kan niet worden bepaald wanneer <omschrijving> in onderzoek staat (route 49i)
+      Gegeven de persoon met burgerservicenummer '000000012' heeft de volgende gegevens
+      | naam                   | waarde             |
+      | voornamen (02.10)      | Junior             |
+      | geslachtsnaam (02.40)  | Test               |
+      | geboortedatum (03.10)  | gisteren - 15 jaar |
+      | geboorteplaats (03.20) | 0518               |
+      | geboorteland (03.30)   | 6030               |
+      En de persoon heeft de volgende 'inschrijving' gegevens
+      | naam                                  | waarde           |
+      | datum eerste inschrijving GBA (68.10) | morgen - 15 jaar |
+      En de persoon is ingeschreven op adres 'A1' met de volgende gegevens
+      | naam                               | waarde                    |
+      | gemeente van inschrijving (09.10)  | 0518                      |
+      | datum aanvang adreshouding (10.30) | morgen - 15 jaar          |
+      | aanduiding in onderzoek (83.10)    | <aanduiding in onderzoek> |
+      | datum ingang onderzoek (83.20)     | 20240701                  |
+      En de persoon heeft een ouder '1' met de volgende gegevens
+      | naam                                               | waarde             |
+      | burgerservicenummer (01.20)                        | 000000024          |
+      | geslachtsnaam (02.40)                              | Check              |
+      | datum ingang familierechtelijke betrekking (62.10) | gisteren - 15 jaar |
+      En de persoon heeft een ouder '2' met de volgende gegevens
+      | naam                                               | waarde             |
+      | burgerservicenummer (01.20)                        | 000000036          |
+      | geslachtsnaam (02.40)                              | Test               |
+      | datum ingang familierechtelijke betrekking (62.10) | gisteren - 15 jaar |
+      Als gezag wordt gezocht met de volgende parameters
+      | naam | waarde    |
+      | bsn  | 000000012 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000012 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam        | waarde                                                                                                      |
+      | type        | GezagNietTeBepalen                                                                                          |
+      | toelichting | gezag is niet te bepalen omdat de volgende relevante gegevens in onderzoek staan: gemeente van inschrijving |
```

Comment:
@Patrick-4488 @CathyDingemanse in geval van gegevens in onderzoek stel ik deze toelichting voor:
`gezag is niet te bepalen omdat de volgende relevante gegevens in onderzoek staan: %s`

Dit is korter dan wat we eerst hadden bedacht (`Uitspraak is gezag niet te bepalen, omdat er bij de gezagbepaling waardes in onderzoek waren gedetecteerd. Bij het bepalen van gezag werd het volgende veld gebruikt dat in onderzoek staat: %s`). En meer in lijn met wat in het antwoordenmodel gebruikt wordt bij ontbrekende gegevens.

Reply:
wanneer we zeker weten dat er altijd exact 1 gegeven in onderzoek wordt genoemd, kan dat ook worden:
`gezag is niet te bepalen omdat het volgende relevante gegeven in onderzoek staat: %s`
of
`gezag is niet te bepalen omdat %s in onderzoek staat`
9bijvoorbeeld "gezag is niet te bepalen omdat gemeente van inschrijving in onderzoek staat"